frontend/src/components/utilities/attemptCliLogin.ts

```diff
@@ -122,17 +120,7 @@ const attemptLogin = async (
                             iv,
                             tag,
                             privateKey
-                          });
-
-                        const userOrgs = await fetchOrganizations();
-                        const orgId = userOrgs[0]._id;
-                        localStorage.setItem("orgData.id", orgId);
```

Comment:
do we no longer need to set the org to localStorage?

Reply:
In the context of the `attemptCliLogin` method specifically, this isn't needed because the user is directed to the "successfully logged in via CLI" page after which they return to their terminal; this was, however, necessary previously for `attemptLogin` method which is similar but for logging in via browser.

Here, we used to fetch user's organizations to navigate to one after logging in. I've replaced all such similar functionality occurrences with one helper function in `Login.utils.tsx`:

```
export const navigateUserToOrg = async (router: NextRouter) => {
    const userOrgs = await fetchOrganizations();

    if (userOrgs.length > 0) {
        // user is part of at least 1 org
        const userOrg = userOrgs[0] && userOrgs[0]._id;
        localStorage.setItem("orgData.id", userOrg);
        router.push(`/org/${userOrg}/overview`);
    } else {
        // user is not part of any org
        router.push("/org/none");
    }
}
```